USING.md

```diff
@@ -1,116 +1,210 @@
+To have an local installation of Zonemaster-CLI follow the
+[installation instruction]. When installed, the examples below can be followed.
 
-### Test reports
 
-The different message levels are CRITICAL, ERROR, WARNING, NOTICE, INFO,
-DEBUG, DEBUG2 and DEBUG3. The default reporting level is NOTICE and higher.
-To change the level of reporting you can use the command line switch
---level=LEVEL. The DEBUG levels are mainly used by developers, and are
-useful to debug the different test cases.
+## Invoking the command line tool
 
-The default level reporting is in plain english, but other output formats
-are also available. The `--lang=raw` option will give you the technical
-language output; instead of english the messages will be displayed as a
-combination of test level and test case message for easy mapping into each
-test case executed by Zonemaster. This means that the output is not
-dependent on any language changes, and better suited for automatic parsing
-of the output.
+The most basic use of the `zonemaster-cli` command is to just test a domain, e.g.
+"zonemaster.net".
 
-For even better automatic parsing of the output, the option to have the
-output reported in JSON might can be used. Use `--lang=json` to have the
-output in JSON format. The JSON format is described in the
-[JSON Output](json-output.md) document.
+With Docker:
+```sh
+docker run -t --rm zonemaster/cli zonemaster.net
+```
 
-More detailed documentation can by found in the [manual page](zonemaster-cli.md)
+With local installation:
+```sh
+zonemaster-cli zonemaster.net
+```
 
-### Advanced use
+The output comes continously as the tests are performed.
+```
+Seconds Level     Message
+======= ========= =======
+  21.39 WARNING   The DNSKEY with tag 54636 uses an algorithm number 5 (RSA/SHA1) which is not recommended to be used.
+  21.80 WARNING   DNSKEY with tag 26280 and using algorithm 5 (RSA/SHA1) has a size (1024) smaller than the recommended one (2048).
+  23.61 NOTICE    SOA 'refresh' value (10800) is less than the recommended one (14400).
+```
+
+The test and output can be modified with different options:
+
+* If your machine is for some reason not configured for use with IPv6 you want to
+  disable the use of IPv6 with the `--no-ipv6` option.
+* If you want to have the test case from which the message is printed then
+  include the `--show-testcase` option.
+* If you want to see the messages translated into another language (see
+  "[Translation]" section below) then include e.g. `--locale da` (Docker) or
+  `--locale da_DK.UTF-8` (local installation).
+
+The same test as above with the three options included:
+
+```sh
+docker run -t --rm zonemaster/cli zonemaster.net --no-ipv6 --show-testcase --locale=da
+```
+```sh
+zonemaster-cli zonemaster.net --no-ipv6 --show-testcase --locale=da_DK.UTF-8
+```
+
+The difference between running `zonemaster-cli` on Docker or local installation
+is invocation string, `docker run -t --rm zonemaster/cli` vs. `zonemaster-cli`.
```

Comment:
`is invocation string` -> `is the invocation string`

Reply:
Fixed.